Doc/library/asyncio-eventloop.rst

```diff
@@ -867,6 +867,10 @@ Handle
       Cancel the call.  If the callback is already canceled or executed,
       this method has no effect.
 
+   .. method:: cancelled()
+
+      Return ``True`` if the call was cancelled.
+
```

Comment:
Please add `.. versionadded:: 3.6`

Reply:
3.7 :)

Reply:
OK, added. I hope it will be backported to 3.6?)

Reply:
No. Asyncio is no longer provisional and this is not a bug fix.